### ISSUE TITLE
Travis fails with Stack 1.3.0

### ISSUE THREAD

**Issue by asr** [agda/agda#2333]
Stack 1.3.0 was [released](https://hackage.haskell.org/package/stack-1.3.0). From its [changelog](https://hackage.haskell.org/package/stack-1.3.0/changelog):

> Stack will now always use its own GHC installation, even when a suitable GHC installation is available on the PATH. To get the old behaviour, use the `--system-ghc` flag or run `stack config set system-ghc --global true`. 

In consequence, Travis is [failing]( https://travis-ci.org/agda/agda/jobs/183249836) with Stack 1.3.0:

```
No compiler found, expected minor version match with ghc-8.0.1 (x86_64) (based on resolver setting in /home/travis/build/agda/agda/stack-8.0.1.yaml).

To install the correct GHC into /home/travis/.stack/programs/x86_64-linux/, try running "stack setup" or use the "--install-ghc" flag. To use your system GHC installation, run "stack config set system-ghc --global true", or use the "--system-ghc" flag.
```
@Blaisorblade, I assigned you this issue.

**Comment by Blaisorblade** [comment]
Thanks, I'll take a look ASAP.

-- 
Paolo Giarrusso
From smartphone, sorry for typos or excessive brevity

Il 12 dic 2016 15:15, "Andrés Sicard-Ramírez" <notifications@github.com> ha
scritto:

**Comment by asr** [comment]
@Blaisorblade, I'm testing the fix (see PR #2336) on Travis.
